sql/core/src/main/scala/org/apache/spark/sql/DataFrameImpl.scala

```diff
@@ -93,7 +93,14 @@ private[sql] class DataFrameImpl protected[sql](
       queryExecution.analyzed.resolve(n.name, sqlContext.analyzer.resolver).get
     }
   }
-
+ 
+  protected[sql] def numericColumns(colNames: String*): Seq[Expression] = {
```

Comment:
Do you mean not to filter columns in  numericColumns but in aggregateNumericColumns? Is it?